q2_feature_table/tests/filter/test_filter_samples.py

```diff
@@ -799,14 +876,41 @@ def test_combine_exclude_ids_and_features_filters(self):
                       ['O1', 'O2'],
                       ['S1', 'S2', 'S3'])
         where = "SampleType='tongue'"
+        with self.assertRaisesRegex(ValueError, 'table is empty'):
+            filter_samples(table,
+                           metadata=metadata,
+                           where=where,
+                           exclude_ids=True,
+                           min_features=1,
+                           max_features=1)
+
+        # allow empty table
         actual = filter_samples(table,
                                 metadata=metadata,
                                 where=where,
                                 exclude_ids=True,
                                 min_features=1,
-                                max_features=1)
+                                max_features=1,
+                                allow_empty_table=True)
         self.assertTrue(actual.is_empty())
 
+    def test_allow_empty_table_true(self):
```

Comment:
This test (and the test below) can be removed since you're testing the True/False cases within all of the tests above.